src/domains/movies.js

```diff
@@ -0,0 +1,72 @@
+const { PrismaClient } = require('@prisma/client');
+const prisma = new PrismaClient();
+
+class MovieDomain {
```

Comment:
ooh, nice! Didn't think of going for classes - it definitely makes for a tidy module exports!